release-team/role-handbooks/communications/templates/writing-buddy-assignment.md

```diff
@@ -0,0 +1,21 @@
+# Writing Buddy Assignement
+
+This is a message to send to the owners of the Feature Blog placeholder PRs to assign them a writing buddy.
+
+```
+Hi @xx 👋 vx.yy Communications team here,
+
+@BUDDY as author of #BUDDY-PR, I'd like you to be a writing buddy for @xx on this PR.
+
+Please:
+
+- Review this PR, paying attention to the guidelines and review hints
```

Comment:
"writing buddy", "guidelines" and "review hints" should link to documentation pages. See [this past comment](https://github.com/kubernetes/website/pull/51426#issuecomment-3046723555) for reference